README.md

```diff
@@ -96,61 +93,67 @@ our [official documentation](https://docs.conjur.org/Latest/en/Content/Developer
 
 To start using the SDK in your applications, create a Client instance and then invoke the API on it:
 
-#### With login ID and password
+#### Define connection parameters
 
-```python3
-#!/usr/bin/env python3
+In order to login to conjur you need to have 5 parameters known from advance.
 
-from conjur import Client
+```
+conjur_url = "https://my_conjur.com"
+account = "my_account"
+username = "user1"
+password = "SomeStr@ngPassword!1"
+ssl_verification_mode = SslVerificationMode.TRUST_STORE
+```
 
-client = Client(conjurrc_data=ConjurrcData(...),
-                ssl_verification_mode=SslVerificationMode.TRUST_STORE,
-                credentials_provider=FileCredentialsProvider(),
-                debug=False)
+#### Define ConjurrcData
 
-print("Setting variable...")
-client.set('conjur/my/variable', 'new value')
+ConjurrcData is a data class containing all the non-credentials connection details.
 
-print("Fetching variable...")
-new_value = client.get('conjur/my/variable')
+`connection_info = ConjurrcData(conjur_url=conjur_url,account=account,cert_file = None)`
 
-print("Variable value is:", new_value.decode('utf-8'))
-```
+* conjur_url - url of conjur server
+* account - the account which we want to connect to
+* cert_file - a path to conjur rootCA file. we need it if we initialize the client in `SslVerificationMode.SELF_SIGN`
+  or `SslVerificationMode.CA_BUNDLE` mode
 
-#### Defining the Conjur server endpoint
+#### Storing credentials
 
-A configuration file called `.conjurrc` is used to hold details required to communicate to the
-Conjur server. You can provide these details needed to open a connection to the Conjur endpoint in
-this file instead of passing them in (`url`, `account`, and `ca_bundle`)  
-during initialization of the Client.
+The client uses credentials provider called `CredentialStores` which inherit from `CredentialsStoreInterface`. This
+approach allow to keep the credentials in a safe location and provide it to the client on demand.
 
-The `.conjurrc` file should be saved to your home directory and should contain `conjur_url`,
-`conjur_account`, and`cert_file`.
+We provide the user with `CredentialStoreFactory` which create such Credential stores.
```

Comment:
who is "the user"? What are we staing here?

Reply:
The user is the reader of the document

Reply:
The SDK's `CredentialStoreFactory` creates such credential stores.